src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -118,4 +222,227 @@ public function iShouldBeNotifiedThatItHasBeenSuccessfullyCreated(): void
     {
         Assert::true($this->responseChecker->isCreationSuccessful($this->client->getLastResponse()));
     }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyEdited(): void
+    {
+        Assert::true(
+            $this->responseChecker->isUpdateSuccessful($this->client->getLastResponse()),
+            'Product could not be edited'
+        );
+    }
+
+    /**
+     * @Then I should be notified that this product is in use and cannot be deleted
+     */
+    public function iShouldBeNotifiedThatThisProductIsInUseAndCannotBeDeleted(): void
+    {
+        Assert::false(
+            $this->responseChecker->isDeletionSuccessful($this->client->getLastResponse()),
+            'Product can be deleted, but it should not'
+        );
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully deleted
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyDeleted(): void
+    {
+        Assert::true(
+            $this->responseChecker->isDeletionSuccessful($this->client->getLastResponse()),
+            'Product still exists, but it should not'
+
+        );
+    }
+
+    /**
+     * @Then I should see :numberOfProducts products in the list
+     */
+    public function iShouldSeeProductsInTheList(int $count): void
+    {
+        Assert::count($this->responseChecker->getCollection($this->client->getLastResponse()), $count);
+    }
+
+    /**
+     * @Then I should( still) see a product with :field :value
+     */
+    public function iShouldSeeProductWith(string $field, string $value): void
+    {
+        Assert::true(
+            $this
+                ->responseChecker
+                ->hasItemWithTranslation($this->client->getLastResponse(), 'en_US', $field, $value),
+            sprintf('Product has not %s with %s', $field, $value)
+        );
+    }
+
+    /**
+     * @Then I should not see any product with :field :value
+     */
+    public function iShouldNotSeeAnyProductWith(string $field, string $value): void
+    {
+        $this->client->update();
+
+        Assert::false(
+            $this
+                ->responseChecker
+                ->hasItemWithTranslation($this->client->getLastResponse(), 'en_US', $field, $value),
```

Comment:
Something seems wrong with this formating
```suggestion
                ->hasItemWithTranslation($this->client->getLastResponse(), 'en_US', $field, $value)
            ,
```